lib/converters/json_to_jdl_entity_converter.js

```diff
@@ -189,7 +189,7 @@ function getRelationship(relationship, entityName) {
     commentInTo: null,
     otherEntity: null,
     toJdlEntity: null,
-    options: new Set()
+    options: []
```

Comment:
why?

Reply:
because `Set` works with shallow equals, therefore adding the same option twicefor non UnaryOptions will pass the test. You could ask why not a Map/Object because the same annotation can be used twice. For places where this was the case, it's still is. for places where I added options, I preferred keeping it simple and directly expose the object.